app/views/mailers/notify_mailer/new_mention_email.html.erb

```diff
@@ -1,5 +1,5 @@
 <h2 style="font-size:25px;">
-  <a href="https://dev.to/<%= @mentioner.username %>"><%= @mentioner.name %></a> just mentioned you in their <%= @mention.mentionable_type.downcase %>!
+  <a href="<%= app_url(@mentioner.username) %>"><%= @mentioner.name %></a> just mentioned you in their <%= @mention.mentionable_type.downcase %>!
```

Comment:
same, if mentioner is a user, you can use `user_url`

Reply:
I need a sanity check here. On dev we use  this 👇🏽  in `app/views/mailers/notify_mailer/new_badge_email.html.erb`
```
<a
      href="<%= user_url(@user) %>"
      style="font-weight:bold;color:white;background:#4e57ef;padding: 5px 10px;border-radius:3px;text-decoration:none">
      Check out your profile
    </a> 
```

Which in development redirects to `http://localhost:3000/users/1` but on prod it works fins and redirects to `https://dev.to/<username>` . I double chceked this on a badge email I have. This is what happens on mentions and follows where I want to use user_url. @rhymes I noticed you wrote this code so maybe you can keep me sane

Reply:
Nvm, figured it :) our the user_url from the application helper is clashing with the user_url from the  Rails.application.routes.url_helpers 🤦🏽‍♀️